pkg/minikube/problem/err_map.go

```diff
@@ -36,6 +36,12 @@ var vmProblems = map[string]match{
 		URL:    "https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver",
 		Issues: []int{1926, 4206},
 	},
+	"OLD_HYPERKIT_VERSION": {
+		Regexp: re(`Add warn about old driver version (hyperkit, kvm)`),
+		Advice: "Too old hyperkit version. Jump forward to new valid hyperkit version as a workaround.",
+		URL:	"https://github.com/kubernetes/minikube/blob/master/docs/drivers.md"
```

Comment:
```suggestion
		URL:	"https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-upgrade",
```

There's a specific link for hyperkit upgrade. Also missing a comma at the end.